### DISCUSSION TITLE
Changelog for released versions?

### DISCUSSION THREAD

**Discussion by TearOfTheStar** [stride3d/stride#2201] [General]
Is there a place where i can read full changelog for every version release? Or at least for latest in-launcher release.

**Comment by VaclavElias** [comment]
Currently, we only have this log for 4.2

https://doc.stride3d.net/latest/en/ReleaseNotes/ReleaseNotes.html#whats-changed-in-details

We would like to introduce the same in a few days also for all updates here https://github.com/stride3d/stride/releases
